contracts/DePayPaymentsV1.sol

```diff
@@ -67,16 +64,16 @@ contract DePayPaymentsV1 is Ownable {
   // In case of ETH we need to deduct what has been payed in as part of the transaction itself.
   function _balanceBefore(address token) private returns (uint balance) {
     balance = _balance(token);
-    if(token == ZERO) { balance -= msg.value; }
+    if(token == Helper.ETH()) { balance -= msg.value; }
   }
 
   // This makes sure that the sender has payed in the token (or ETH)
   // required to perform the payment.
   function _ensureTransferIn(address tokenIn, uint amountIn) private {
-    if(tokenIn == ZERO) { 
+    if(tokenIn == Helper.ETH()) { 
```

Comment:
`tokenIn == Helper.ETH()` It's cost too much please consider to use local const.

Reply:
We moved it back into local variables for each contract that relies on it (to save some gas).
```
 uint public constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
 ```

Not enough to form a BaseContract just yet 

🙏